includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1148,7 +1148,9 @@ public function get_onboarding_business_types() {
 		return $this->request(
 			[],
 			self::ONBOARDING_API . '/business_types',
-			self::GET
+			self::GET,
+			true,
+			true
```

Comment:
maybe it would be a good idea to add a test that checks that the method was called with user_token = true, won't be easy to break it again :)

Reply:
Addressed in 0df59ed